daemon/cmd/cleanup.go

```diff
@@ -83,6 +83,7 @@ func (d *daemonCleanup) registerSigHandler() <-chan struct{} {
 			pidfile.Clean()
 			d.Clean()
 			d.cleanupFuncs.Run()
+			// nolint
```

Comment:
What is this comment for?

Reply:
@qmonnet ideally, _break_ should not be called directly inside for loop (e.g. only the first iteration is run, and it's not really for loop). But I notice that this is intentional, so just add nolint. I can refactor the code a little bit to avoid this _break_. What do you have in mind ?

Reply:
Nothing particular in mind, I just had no idea what the linter would say here, thanks for the explanation :). No preference on refactoring, it would probably be good but doesn't have to be part of this PR.